### ISSUE TITLE
Sentient Items Antagonists

### ISSUE THREAD

**Issue by YogsMcree** [yogstation13/yogstation#1690]
So, I was looking at old suggestions and saw the "sentient sword" suggestion. Why, exactly, do more things like this not exist? Why does every antangonist have to be a walking, talking, convert or kill menace? Why cant we have Traitor + Antag items game mode? Or just Antag Items added to secret rotation? These items could be weapons, armor or just clothing that have the ability to mentally contact anyone around them. A book on a bookshelf whispers to you "Come here nerd. Pick me up." If you choose to do so, you gain an objective and a power based on the item (Wizard spell or two for a book, a shield for weapons or staves, obviously item specific choices.) The objective can be "Save the station from Evil", or it could be "Descend the station into chaos", "Murder Bob Blitzer, but do not get caught.", "Assist the Clown in his objectives." etc. This would add some nice spice every few rounds, instead of OMG ALIUMS, OMG REVS, OMG GANGS.

**Comment by Lightning38088** [comment]
HA murder bob blitzer? good luck.

**Comment by YogsMcree** [comment]
That was the point :^)

**Comment by Time-Green** [comment]
Bob's weakness is a bee suit.

**Comment by YogsMcree** [comment]
You know, I was there for the beeman round. But I was a scientist and missed all the antics.

**Comment by Dagdammit** [comment]
What if you gave a melee implement some magic spells (that can either target the wielder or itself, the latter just charging it the same way a touch spell charges the hand, so the spell's applied to the next the weapon strikes), the ability to speak, and then made it effectively a silicon with (say) the paladin lawset. So it still HAS to follow its laws, but it has a completely different kind of power to exercise in following them.

**Comment by YogsMcree** [comment]
I mean, that leads to a conflict of interest between a free human (wielder) and the wielded. Although the power to DENY usage of the wielder would be cool.

**Comment by Dagdammit** [comment]
I've had some thoughts on using intent in the past.

Help intent: Mercy. About half your normal damage is instead stamina damage.
Disarm intent: Defy. 20% chance for the wielder to fumble their attack and drop you with each strike.
Aggressive intent: Kill. Your base damage raises by 5, maybe 10.
Grab intent: Bind. You become a nodrop item that the wielder can't put down.

**Comment by AdamElTablawy** [comment]
Again a stupid idea that simply won't happen.

**Comment by Rafael450** [comment]
I like it

**Comment by YogsMcree** [comment]
Oh look, Adam says no and literally doesn't give anything constructive again. What a surprise.

**Comment by YogsMcree** [comment]
How about a why Adam. Can you be asked to defend your opinion

**Comment by Dagdammit** [comment]
I mean there isn't nearly enough going on in this concept to possibly work as more than like, a random midround event kind of deal. Possessed Blade is already a chaplain null rod choice, the darn thing just currently can't do anything besides speak.

**Comment by puremanbird** [comment]
its a bad idea for many obvious reasons. it encourages the wielder to break the rules. its fine for a possessed blade to encourage bloodshed rp wise but its not ok for the chaplain/wielder to go around chopping up random people. i know we're low-mid rp but just no. 

its good that your trying to help the server with ideas but please think through them more than just spouting off the first thing on the top of your head.

**Comment by Dagdammit** [comment]
Oh god no, it'd be a horrible thing to do for the chaplain weapon. This was
if it was going to somehow be a midround antag spawn like the revenant.

**Comment by YogsMcree** [comment]
Thank you pure for an actually constructive decline

**Comment by doerakpoes** [comment]
So it sounds good but, they have to lose their objective and powers if they lose the item.
Or they keep the objective and powers but the item won't give the objective and powers to another being.

**Comment by doerakpoes** [comment]
Wait i just got it instead of kill em objectives.
Protect em objectives with powers A FRIENDLY EVENT instead of killing.

**Comment by YogsMcree** [comment]
I mean even attainevents. Maybe the inital creator can track the item across the universe and wants the captains hand tele, or a being with telekinesis. I think its a really high potential idea. But the coders dont seem to like it.

**Comment by Cruix** [comment]
Pureman and Adam hate almost everything, and pure isn't even a coder. Don't let them drag you down too much.

**Comment by YogsMcree** [comment]
I mean I am also not a coder, so I cant bring the idea to fruition. But if a code says "tell me what, and make sure its balanced and cleared with council" i would be more than willing to create a more in depth idea.

**Comment by AdamElTablawy** [comment]
Here's my reasoning.

The idea and execution is terrible.

First off, an item that gives you free antag with no downsides. Yay, fantastic. Everyone will grab them immediately and go do the usual thing of kill all the 'allowed' valids-all heads, all sec, your target, anyone who fights you, etc

Wizard spells are intentionally overpowered, giving them to non-wizards is a bad fucking idea. See-Even fireball can kill a whole fuckton of people if used right.

It would also just be an OMG ANTAG ITEMS instead of OMG gangs, etc


I can see the point of wanting to add a new antagonist, but adding a sapient item is...

Well first off it's dumb in concept. Every other antagonist has some sort of reason for existing, some sort of relation to any sort of weird sci-fi theme. Changelings are the thing/prototype, Traitors are obvious, nuclear operatives are fairly obvious, shadowlings are aliens, cultists fit an eldritch horror theme well...

And then there's a book that tells you to kill people. What the fuck?

Additionally, might I point out that as a roundstart antagonist this would fucking suck? You're immobile and can't move and you're stuck in place in the hopes someone picks you up. It'd be boring as hell and you're essentially the chaplain's sword, which is fairly limited fun, and out of all antagonists you could possibly get you're cucked into getting this one? 

Not to mention it'd be easy to meta when it occurs, AND if they occur absolutely on any gamemode it'd be a nightmare for both the administration team and the crew. Security has to deal with some murderboning cunt with wizard spells or genetic powers or whatever this dumb thing gives you and the antag of the round, and for the admin team they have to deal with shit like 'fellow cultist intentionally grabs possessed items for shit, gets defend station objectives, murders other cultists with cult powers.'

Not to mention that it could be considered self-antagging along the same lines of intentionally walking onto a sigil or something to be converted to grab a possessed item, except even more so because possessed weapons literally have nothing to offer aside from free antag, unlike any convert antag that can try to persuade, etc

To top it off, how do you manage this? Does it occur on lowpop? Hard to make it NOT, if it occurs in any gamemode as a roundstart antagonist. Pop requirement is in the gamemode code itself for the roundtype, not the antagonist of the roundtype. 

Fucks up any hope of datum antags as well, so that's fun.

If it's not a roundstart antagonist and it's a random event that's still bad because of all of the above and pop requirements still, AND to top it off now you're offering more ghosts ways back into the round (bad) and more antags to fight in a random event list that is basically already 'more antags show up' AND just for the diarrhea icing on the shit cake gives salty antag ghosts free reign to encourage people to kill the guy who killed them/meta about it, and it's not bannable either because even if they didn't remember their past life, hey, complete and utter free reign to kill whoever they want yay better tell the whole station about traitor mc gillicucky

And giving it fucking laws is a terrible idea, Dagdammit.

Now you have something that is literally obligated to hunt antags and gives the fucking person holding it antag, so that's stupidly contradictory, and rips any semblance of 'fun' that the joy of being an immobile powerless object might have right away. Plus, you know, meta, hunt traitor, etc.


As for 'friendly events', Doreakpoes please just stop. No. There are no 'friendly' random events. Random events exist to make a bad situation worse. If this showed up as one, though, it'd be bad for all the above reasons.

Tl;DR shit idea, shit execution, bad antagonist, bad suggestions. 

Are you happy now that I've thoroughly shat all over it or do you want me to go into more detail about how horribly retarded this is?

Hell, do you know how stupid it even sounds? Think about what you're saying. Sapient items antagonists. Doesn't that make you cringe, just a little bit?

**Comment by Dagdammit** [comment]
"Something that is literally obligated to hunt antags and give the person holding it antag."

...The idea of a sword whose code obligates it to push its wielder to take actions AND obligates it to turn against said wielder BECAUSE they took those actions is... fascinating. Like, my wielder is innocent so I want them to use me in self defense so I can defend the innocent. But once that happens my wielder has stabbed people with a sword, and while he's not evil I no longer consider him innocent. As someone who likes exploring+RPing conflicts and contradictions with lawsets (plus the challenge of trying to work with an entity who follows a lawset), that honestly makes me want to have this experience even more.

At the same time you do have a damn good point about how it could suck. It's the thing where you go straight to how stupid people would do stupid things and people would whine about it, whereas I go straight to how it could be cool though.

**Comment by AdamElTablawy** [comment]
YOU will use it to roleplay

Most people will just be dicks

See: borers

**Comment by doerakpoes** [comment]
Did any read my idea or isnit not gud? :(

**Comment by AdamElTablawy** [comment]
it's bad

**Comment by doerakpoes** [comment]
;-; R.I.P. doerakpoes' idea 2017-2017
